plugins/source/aws/resources/services/costexplorer/cost_custom.go

```diff
@@ -0,0 +1,115 @@
+package costexplorer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/costexplorer"
+	"github.com/aws/aws-sdk-go-v2/service/costexplorer/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+	cqtypes "github.com/cloudquery/plugin-sdk/v3/types"
+	"github.com/mitchellh/hashstructure/v2"
+)
+
+var tableName = "aws_alpha_costexplorer_cost_custom"
+
+func CustomCost() *schema.Table {
+	return &schema.Table{
+		Name:     tableName,
+		Resolver: fetchCustom,
+		Title:    "AWS Cost Explorer costs based on custom inputs",
```

Comment:
```suggestion
		Title:    "AWS Cost Explorer costs based on custom inputs (Alpha)",
```

This change would also affect the `This table shows data for...` entry in the markdown and might look a bit different, but I don't think it's a problem.